components/header/Searchbar.tsx

```diff
@@ -1,34 +1,20 @@
-import { headerHeight } from "../../components/header/constants.ts";
 import Searchbar, {
   Props as SearchbarProps,
 } from "../../components/search/Searchbar.tsx";
-import Modal from "../../components/ui/Modal.tsx";
-import { useUI } from "../../sdk/useUI.ts";
 
 export interface Props {
   searchbar?: SearchbarProps;
 }
 
 function SearchbarModal({ searchbar }: Props) {
-  const { displaySearchPopup } = useUI();
-
   if (!searchbar) {
     return null;
   }
 
   return (
```

Comment:
A condicional para o o retorno do componente pode ser dessa forma:
```suggestion
  return searchbar &&(
```
ou também:
```suggestion
  return searchbar ? (
  ...
  ) : null
```

Reply:
@vmedeirossocial Corrigido